tests/test_io_obj.py

```diff
@@ -895,6 +895,52 @@ def check_item(x, y):
         with self.assertRaisesRegex(ValueError, "same type of texture"):
             join_meshes_as_batch([mesh_atlas, mesh_rgb, mesh_atlas])
 
+    def test_save_obj_with_normal(self):
+        verts = torch.tensor(
+            [[0.01, 0.2, 0.301], [0.2, 0.03, 0.408], [0.3, 0.4, 0.05], [0.6, 0.7, 0.8]],
+            dtype=torch.float32,
+        )
+        faces = torch.tensor(
+            [[0, 2, 1], [0, 1, 2], [3, 2, 1], [3, 1, 0]], dtype=torch.int64
+        )
+        verts_normals = torch.tensor(
+            [[0.02, 0.5, 0.73], [0.3, 0.03, 0.361], [0.32, 0.12, 0.47], [0.36, 0.17, 0.9]],
+            dtype=torch.float32,
+        )
+        faces_normals = faces
```

Comment:
perhaps `faces_normals = faces * 3` or something to make it more obvious that the format is correct.

Reply:
updated the test with more normals per face